docs/api.md

```diff
@@ -29,18 +29,18 @@ localForage includes a localStorage-backed fallback store for browsers with no I
 
 ```bash
 # Install via npm:
-npm install localforage
+npm i localforage
 
-# Or with bower:
-bower install localforage
+# Or, with yarn:
+yarn add localforage
 ```
 
 ```html
 <script src="localforage.js"></script>
 <script>console.log('localforage is: ', localforage);</script>
 ```
 
-To use localForage, [download the latest release](https://github.com/mozilla/localForage/releases) or install with [npm](https://www.npmjs.org/) (`npm install localforage`) or [bower](http://bower.io/) (`bower install localforage`).
+To use localForage, [download the latest release](https://github.com/localForage/localForage/releases) or install with [npm](https://npmjs.org/) (`npm i localforage`), or [yarn](https://yarnpkg.com) (`yarn add localforage`)
```

Comment:
Looks good, but could you change the shorthand back from `i` to `install`? It makes it a lot easier to read 😄